src/Framework/AspNetCoreAnalyzers/src/Analyzers/RouteHandlers/RouteHandlerAnalyzer.cs

```diff
@@ -71,6 +91,8 @@ public override void Initialize(AnalysisContext context)
                     return;
                 }
 
+                mapOperations.Add(new MapOperation(invocation, routeUsage));
```

Comment:
Where do you handle bailing out from block analysis if any invocation is within a conditional or loop statement?